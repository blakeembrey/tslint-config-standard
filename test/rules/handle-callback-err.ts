export function cb (err) {
  console.log('hello world')
}
