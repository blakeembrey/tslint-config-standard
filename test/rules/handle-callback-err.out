handle-callback-err.ts[1, 14]: Expected error to be handled
